build/build_windows.ps1

```diff
@@ -0,0 +1,30 @@
+if ($args -contains "-it") {
+    if (-not (Get-Command git -ErrorAction SilentlyContinue)) {
+        winget install --id Git.Git -e
+    }
+    if (-not (Get-Command python -ErrorAction SilentlyContinue)) {
+        winget install --id Python.Python -e
+    }
+    if (-not (Get-Command cl -ErrorAction SilentlyContinue)) {
```

Comment:
This will trigger the `winget` installation when a VS CLI session is not active, which is probably not what you want (given that `scons` is able to figure out where it's at without a VS command prompt window not needing to be active).